plinth/utils.py

```diff
@@ -52,6 +52,11 @@ def non_admin_view(func):
     return func
 
 
-def is_user_admin(user):
+def is_user_admin(request, cached=False):
     """Return whether user is an administrator."""
-    return user.groups.filter(name='admin').exists()
+    if cached and request.session['cache_user_is_admin'] is not None:
```

Comment:
However without caching, 
I get 2 queries each time I make a request, so I am assuming caching is working. Updating my PR. :smile: 

![image](https://cloud.githubusercontent.com/assets/20185076/25006414/6ab98a50-207a-11e7-8db4-8a84aed676bb.png)